LEGO1/lego/legoomni/include/registrationbook.h

```diff
@@ -33,6 +33,20 @@ class RegistrationBook : public LegoWorld {
 
 	// SYNTHETIC: LEGO1 0x10076f30
 	// RegistrationBook::`scalar deleting destructor'
+private:
+	undefined4 m_unk0xf8;  // 0xf8
+	undefined m_unk0xfc;  // 0xfc
+	undefined2 m_unk0x28e;  // 0x28e
+	undefined4 m_unk0x280;  // 0x280
+	undefined4 m_unk0x284;  // 0x284
+	undefined4 m_unk0x288;  // 0x288
+	undefined2 m_unk0x28c;  // 0x28c
+	undefined2 m_unk0x2b8;  // 0x2b8
+	undefined4 m_unk0x2bc;  // 0x2bc
+	undefined m_unk0x2c1;  // 0x2c1
+	undefined4 m_unk0x2c4;  // 0x2c4
+	undefined4 m_unk0x2c8;  // 0x2c8
+	undefined4 m_unk0x2cc;  // 0x2cc
```

Comment:
This isn't how member allocation works. The compiler does not make decisions based on the labels of the members of a given class. You're going to have to manually add padding to line these up by calculating the difference between the offsets that are already present and in-place and the ones you need for the constructor.

Reply:
Sorry, what I need to do? Because I don't know the structs.